.github/workflows/create_llvm.yml

```diff
@@ -18,20 +18,26 @@ jobs:
         version: [17, 18]
         os: [ubuntu-22.04, windows-2019]
         build_type: [Release, RelAssert]
-        arch : [x86_64, aarch64]
+        arch : [x86, x86_64, aarch64]
         exclude:
-          # Reduce the versions of llvm for aarch64 architecture, as
+          # Reduce the versions of llvm for aarch64 & x86 architectures, as
```

Comment:
windows too, not part of this change as such, but since you are changing the line.

Reply:
Done